### ISSUE TITLE
Proper pub points not given

### ISSUE THREAD

**Issue by quetool** [dart-lang/pub-dev#7129]
URL: https://pub.dev/packages/web3modal_flutter/versions/3.0.0-beta04

Hello, on the package linked above☝️  there is a proper LICENSE file filled with an OSI-approved License however it's not being recognized as such.

**Comment by quetool** [comment]
We can see how `dio_cache_interceptor` package is using the very same LICENSE as we do but it's being recognized properly for them but not for our package https://pub.dev/packages/dio_cache_interceptor/license

**Comment by quetool** [comment]
Only after hitting documentation points on latest version the license was recognized. Not sure if it's a bug or not but IMHO license recognition should be independent from documentation analysis. 

https://pub.dev/packages/web3modal_flutter/versions/3.0.0-beta04/score

Vs

https://pub.dev/packages/web3modal_flutter/versions/3.0.0-beta06/score

**Comment by isoos** [comment]
@quetool: thanks for reporting! A few notes:
- license recognition **is** independent from documentation generation, although both happen as an analysis/worker step (and subject to change over time as we upgrade our dependent libraries)
- license detection is not an exact science, small difference may change the meaning of the license in significant ways, we rely on similarity threshold to match common license texts
- you did change the license in a non-insignificant way to pass the detection: https://github.com/WalletConnect/Web3ModalFlutter/commit/75761ab5cfc9a9d627283ed291d7e8e6bc4842fb#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7
  I've filed https://github.com/dart-lang/pana/issues/1268 for later investigation

**Comment by quetool** [comment]
Thanks for answering @isoos !
Indeed I figured "hey, maybe the appendix is important for the recognition tool" and I added it, although I would have never said it :)
Didn't know it was that important

**Comment by isoos** [comment]
It is not the appendix that is important, rather, overall, the difference from the "canonical" (or one of the many canonical) text content. Putting the appendix back moved the license over the (otherwise strict) detection threshold. It is important to report these issues, otherwise fixing these won't get a high priority in the near future...

**Comment by sigurdm** [comment]
I don't think we need to update here. It seems to me that this is a (potential) significant change to the licence file, and I'm happy we detect it.

Closing for now.
